commit 40a554d5ca9c08a4dd3181bb47339aded5252628
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Thu Jan 20 03:32:35 2022 +0100

    disable ppc64le asm
    
    This is problematic in threaded applications with musl, so
    just fall back to plain-C version.

diff --git a/meson.build b/meson.build
index bd54e78..eafdfad 100644
--- a/meson.build
+++ b/meson.build
@@ -1346,11 +1346,6 @@ elif host_machine.cpu_family() == 'sparc64'
     with_asm_arch = 'sparc'
     pre_args += ['-DUSE_SPARC_ASM']
   endif
-elif host_machine.cpu_family() == 'ppc64' and host_machine.endian() == 'little'
-  if system_has_kms_drm
-    with_asm_arch = 'ppc64le'
-    pre_args += ['-DUSE_PPC64LE_ASM']
-  endif
 elif host_machine.cpu_family() == 'mips64' and host_machine.endian() == 'little'
   if system_has_kms_drm
     with_asm_arch = 'mips64el'
